governance/polls/Ratification Poll for MIP55 Amendments (MIP4c2-SP16) - February 14, 2022.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for MIP55 Amendments (MIP4c2-SP16) - February 14, 2022
+summary: This amendment for MIP55: Special Purpose Fund aims to improve the clarity of the original MIP text
+discussion_link: https://forum.makerdao.com/t/mip4c2-sp16-mip55-amendments/12606
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-02-14T16:00:00
+end_date: 2022-02-28T16:00:00
+---
+# Ratification Poll for MIP55 Amendments (MIP4c2-SP16) - February 14, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, February 14 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP4c2-SP16 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This amendment for MIP55: Special Purpose Fund aims to improve the clarity of the original MIP text, allowing a clear distinction between which MIP55c3 subproposal may enter the Weekly Cycle and which has to enter the Monthly Governance Cycle, their respective Feedback and Frozen Periods, and giving much needed clarification for proposals of exactly 50k DAI. This amendment also brings some minor typo fixes to MIP55c1 and MIP55c2.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/e1e6659da4b33a4dadcf24ed762151e52b14b08d/MIP4/MIP4c2-Subproposals/MIP4c2-SP16.md)
+* [Latest Proposal Version]($link_to_portal_version)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip4c2-sp16-mip55-amendments/12606)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
+* The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on Thursday, February 24. 
```

Comment:
The date here is wrong, the vote doesn't finish until the 28th, so it would be confirmed on Thursday, March 3rd